src/screens/HomePage/HomePage.style.tsx

```diff
@@ -18,16 +18,42 @@ export const Content = styled.div`
   margin: auto auto 3rem;
 `;
 
-export const Section = styled.div`
+export const HomePageBlock = styled.div`
```

Comment:
Renamed Section => HomePageBlock because we also have `<SectionWrapper>` and a location page specific `<Section>` which seemed confusing...